### DISCUSSION TITLE
Personalize generated file names in modular mode

### DISCUSSION THREAD

**Discussion by relu91** [acacode/swagger-typescript-api#786] [Q&A]
Hi! thank you for making this library. I'm evaluating it for one of our projects. Currently, we configured it with the --modular option enabled, but we were wondering how we can persolize a little bit more the output. In particular, it seems that the library generates `Endpoint.ts` files one for each swagger tags. Is there a function where we can add prefixes or suffixes to this name? Can I also change it from Pascal case to camelCase?

**Comment by Grawl** [comment]
Can you hint me how to turn on modular mode?

**Reply by relu91** [reply]
When you call the `generateApi` function, you can add `modular: true`. 
```ts
  await generateApi({
     // ... other parameters
     modular: true
  })
```
